index.js

```diff
@@ -113,7 +113,11 @@ exports.fromMapFileComment = function (comment, dir) {
 
 // Finds last sourcemap comment in file or returns null if none was found
 exports.fromSource = function (content, largeSource) {
-  if (largeSource) return convertFromLargeSource(content);
+  if (largeSource) {
+    var res = convertFromLargeSource(content);
+    if (typeof res === 'undefined') res = null;
+    return res;
```

Comment:
well you need three lines if you test for undefined, no?
```
if (largeSource) {
  var res = convertFromLargeSource(content);
  if (typeof res === 'undefined') return null;
  return res;
}
```
which is no or less readable - and a personal preference of mine is to keep the number of return statements to a minimum (goal of 1 save for validations at the beginning).

Just explaining my reasoning.  I understand it's negligible and that also it's your codebase which I respect.